control/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Converter(models.Model):
+    unit_choices = [("metre", "Metre"),
+                    ("kilometre", "Kilometre"),
+                    ("centimetre", "Centimetre"),
+                    ("millimetre", "Millimetre"),
+                    ("micrometre", "Micrometre"),
+                    ("nanometre", "Nanometre"),
+                    ("mile", "Mile"),
+                    ("yard", "Yard"),
+                    ("foot", "Foot"),
+                    ("inch", "Inch")]
+
+    unit = models.CharField(null=True, blank=True, max_length=15, choices=unit_choices)
+
+    unit_to = models.CharField(null=True, blank=True, max_length=15, choices=unit_choices)
+
+    unit_value = models.FloatField(null=True, blank=True)
+
+    def unit_converter(self):
```

Comment:
Esto realmente no deberia estar aqui, de hecho ni siquiera necesitas el modelo para calcular, sino que lo "necesitas" para guardar records (por ejemplo). Yo sacaria de aqui esta funcion y en lugar de `self` como atributo de entrada, pondria `unit_value`, `unit` y `unit_to`, que es lo que vas a sacar del form. Does that make sense?